src/components/Pill/__tests__/Pill.spec.js

```diff
@@ -1,21 +1,51 @@
 /**
  * @file Pill tests.
- * @copyright IBM Security 2019
+ * @copyright IBM Security 2020
  */
 
-import { shallow } from 'enzyme';
+import { render } from '@testing-library/react';
 import React from 'react';
+import userEvent from '@testing-library/user-event';
 
-import Pill from '../';
+import { Pill } from '../../..';
 
-describe('Pill', () => {
-  let pill;
+describe('PanelV2', () => {
+  test('should have no Axe or DAP violations', async () => {
+    const main = document.createElement('main');
+    render(<Pill value="127.0.0.1" type="IP" />, {
+      // DAP requires a landmark '<main>' in the DOM:
+      container: document.body.appendChild(main),
+    });
+    await expect(document.body).toHaveNoAxeViolations();
+    await expect(document.body).toHaveNoDAPViolations('Pill');
+  });
+
+  test('should add a custom class', () => {
+    const { getByText } = render(
+      <Pill value="127.0.0.1" type="IP" className="custom-class" />
+    );
 
-  beforeEach(() => {
-    pill = shallow(<Pill value="127.0.0.1" type="IP" />);
+    expect(getByText(/127.0.0.1/i).parentNode).toHaveClass('custom-class');
   });
 
-  it('renders correctly', () => {
-    expect(pill).toMatchSnapshot();
+  test('should be inert and non-interactive', () => {
+    const { getByText } = render(
+      <div>
+        <Pill value="127.0.0.1" type="IP" />
+        <button>test button</button>
+      </div>
+    );
+
+    userEvent.tab();
+
+    // Expect only the extra test button to be interactive:
+    expect(getByText(/127.0.0.1/i).parentNode).not.toHaveFocus();
+    expect(getByText(/test button/i)).toHaveFocus();
```

Comment:
Great! I think we can end the assertion here 💯

Reply:
Hmm I don't agree. How can we be sure that order isn't at play here somehow, unless we do a complete loop?

Reply:
Would you mind elaborating on what you mean by the order? Initially, this felt redundant due to the `button` being placed after the `Pill` in the render.

Reply:
Honestly not sure what I meant here. 👍 I'll remove it.